.env.example

```
@@ -73,7 +73,9 @@ export PROMO_LIST={}
 export NEXT_PUBLIC_VAPID_PUBLIC_KEY=
 export VAPID_PRIVATE_KEY=
 export NEXT_PUBLIC_VAPID_SUBJECT="mailto:hello@peanut.me"
-export NEXT_PUBLIC_FETCH_TIMEOUT_MS=10000
+# Overrides BOTH fetch budgets (client 20s / server 30s, src/utils/sentry.utils.ts).
+# Positive integer of milliseconds; empty or malformed = keep the defaults.
+export NEXT_PUBLIC_FETCH_TIMEOUT_MS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_FETCH_TIMEOUT_MS key should go before the NEXT_PUBLIC_VAPID_PUBLIC_KEY key

(UnorderedKey)